src/components/Pages/Location/LocationServiceList.js

```diff
@@ -4,23 +4,26 @@ import { find, capitalize } from 'lodash';
 import { useIntl } from 'react-intl';
 
 import { getDaysInOrder } from 'js/helpers/date';
-import { date as i18nDate, locations as i18nLocations } from 'js/i18n/definitions';
+import {
+  date as i18nDate,
+  locations as i18nLocations,
+} from 'js/i18n/definitions';
 
-const DayHours = ({day, hours}) => {
+const DayHours = ({ day, hours }) => {
   const intl = useIntl();
   return (
     <tr>
       <td className="coa-LocationPage__table-service-label">
-        {intl.formatMessage(
-          i18nDate['weekday' + capitalize(day)],
-        )}
+        {intl.formatMessage(i18nDate['weekday' + capitalize(day)])}
+      </td>
+      <td>
+        {hours !== null ? hours : intl.formatMessage(i18nLocations.closed)}
```

Comment:
instead of setting hours to "Closed" in cleanData, we just set it to null and localize here